### DISCUSSION TITLE
86Box Source Code is uncompatible with Cmake standard installation

### DISCUSSION THREAD

**Discussion by Pellzi** [86Box/86Box#4959] [Q&A] [answered]
I tryed out to compile the actual source of 86Box with cmake, but compilation failed.
I have a standard Win10x64 Home (22H2), standard Vs2022 (17.11.5), and a standard Cmake (3.30.5) Installation.
The Output from Cmakes looks as follow:

![Error-during Compiling](https://github.com/user-attachments/assets/abaa8c40-6ae6-499b-b7a9-5c18b0fc0514)

What steps i have to take, that i can sucessfully compile this project?

**Comment by jriwanek** [comment, marked as answer]
https://86box.readthedocs.io/en/latest/dev/buildguide.html

**Reply by Pellzi** [reply]
This answer is nearly compatible to the Text written in Row 259 of src/crcspeed/crc64speed.c
"deserunt mollit anim id est laborum" -> "that is, they have to toil.
(If the meaning of this text is other, then the translator from Latin has made a fault.)

**Comment by OBattler** [comment]
You're missing the FreeType library, so install it.

**Reply by OBattler** [reply]
We also don't support Microsoft's C compiler.

**Reply by Pellzi** [reply]
On the way doing this, a question to you:
When have you the last time updated all the necessary dependencies to build 86Box?
[Dependencies for building 86Box.txt](https://github.com/user-attachments/files/17718444/Dependencies.for.building.86Box.txt)
